src/css/silex.scss

```diff
@@ -23,6 +23,13 @@
     font-size: 18px;
     font-weight: normal;
   }
+  //Better contrast than grapesjs default for select menu et input
+  select, input {
+    background-color: $darkerPrimaryColor;
+  }
+  .gjs-color-warn, .gjs-sm-sector .gjs-sm-field input, .gjs-clm-tags .gjs-sm-field input, .gjs-sm-sector .gjs-clm-field input, .gjs-clm-tags .gjs-clm-field input, .gjs-sm-sector .gjs-sm-field select, .gjs-clm-tags .gjs-sm-field select, .gjs-sm-sector .gjs-clm-field select, .gjs-clm-tags .gjs-clm-field select {
+    color: #ddd;
```

Comment:
Here you should use one of the variables for colors
Maybe just make the dark variable darker?

Reply:
oh yes you're right, thx